### PR TITLE
README: download the source first to use rust-version

### PR DESCRIPTION
**Pull request by rubo77**
At this point, the file `rust-version` is not present yet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Make sure you have at least 1.5 GB free disk space.
 
     ```
     curl https://sh.rustup.rs -sSf > rustup.sh
-    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat rust-version)
+    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(curl https://raw.githubusercontent.com/freifunk-saar/ff-node-monitor/master/rust-version)
```
Comment (RalfJung) [inline README.md:39]: Good catch!

But this step doesn't seem right... IMO it makes more sense to just fetch the sources first.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,6 @@ Make sure you have at least 1.5 GB free disk space.
 
     ```
     sudo adduser ff-node-monitor --home /opt/ff-node-monitor --system
-    cd /opt/ff-node-monitor
```
Comment (RalfJung) [inline README.md:25]: Why did you move this?

File: README.md [hunk 3]
```diff
@@ -31,31 +30,36 @@ Make sure you have at least 1.5 GB free disk space.
...
-    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat rust-version)
+    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat src/rust-version)
     rm rustup.sh
     ```
 
     The file `rust-version` always contains a tested nightly version number. If
     you want the latest nightly version instead, just use `--default-toolchain nightly` .
 
-4.  Now we can fetch the sources and build them:
+5.  Start the build process:
```
Comment (RalfJung) [inline README.md:52]:
```suggestion
5.  Build node-monitor:
```

File: README.md (hunk 2 above)
Comment (rubo77) [reply README.md:25]: I had to

Comment (rubo77) [reply README.md:25]: So it is more obvious, that rust is installed one directory above the git repository

Comment (rubo77) [reply README.md:25]: And it belongs logically to the rust install step

File: README.md [hunk 4]
```diff
@@ -31,31 +30,36 @@ Make sure you have at least 1.5 GB free disk space.
...
+    ```
+
+4.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
     [Rocket](https://rocket.rs/), which means it needs a nightly version of Rust:
 
     ```
+    cd /opt/ff-node-monitor
     curl https://sh.rustup.rs -sSf > rustup.sh
-    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat rust-version)
+    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat src/rust-version)
```
Comment (RalfJung) [inline README.md:45]: This is really odd and not obvious at all though, you previous referred to this directory as `/opt/ff-node-monitor/src` and now you use the relative path.

File: README.md [hunk 5]
```diff
@@ -31,31 +30,36 @@ Make sure you have at least 1.5 GB free disk space.
     sudo apt install libssl-dev libpq-dev libc6-dev curl gcc pkg-config
     ```
 
-3.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
+3.  Fetch the sources:
+    
+    ```
+    sudo -u ff-node-monitor git clone https://github.com/freifunk-saar/ff-node-monitor.git /opt/ff-node-monitor/src
```
Comment (RalfJung) [inline README.md:36]: Please use `~ff-node-monitor` instead of `/opt/ff-node-monitor`.

File: README.md (hunk 4 above)
Comment (rubo77) [reply README.md:45]: But I change directly in line 37 : cd /opt/ff-node-monitor
so it is clear, that all in this **step 4.** has this directory as base.

My idea was mainly, that each single step is logical in itself, in case you don't do all steps directly after-another.

File: README.md (hunk 5 above)
Comment (rubo77) [reply README.md:36]: I think, we had this earlyer, and decided against the notation `~ff-node-monitor`, Maybe it was not working on all systems like this?

Comment (rubo77) [reply README.md:36]: You accepted the change in #46. I personally don't think the home-dir writing is so clear, you have to think around the corner, where the user has its home dir and you only save 4 characters anyway

Comment (RalfJung) [reply README.md:36]: We are using `~ff-node-monitor` in some places though, and IMO it is clearer than `/opt`.

But I think it is easier if I just edit the README the way I like it, rather than going back and forth here.^^ I'll do that once I find some time.
